ironfish-cli/src/commands/service/snapshot.ts

```diff
@@ -1,26 +1,17 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
-import {
-  AbortMultipartUploadCommand,
-  CompleteMultipartUploadCommand,
-  CreateMultipartUploadCommand,
-  S3Client,
-  UploadPartCommand,
-} from '@aws-sdk/client-s3'
-import { Assert, FileUtils, NodeUtils } from '@ironfish/sdk'
+import { S3Client } from '@aws-sdk/client-s3'
+import { FileUtils, NodeUtils } from '@ironfish/sdk'
 import { CliUx, Flags } from '@oclif/core'
 import crypto from 'crypto'
 import fsAsync from 'fs/promises'
 import path from 'path'
-import tar from 'tar'
 import { IronfishCommand } from '../../command'
 import { LocalFlags } from '../../flags'
-import { DEFAULT_SNAPSHOT_BUCKET, SnapshotManifest } from '../../utils'
+import { DEFAULT_SNAPSHOT_BUCKET, S3Utils, SnapshotManifest } from '../../utils'
+import { TarUtils } from '../../utils/tar'
```

Comment:
nit - this could be from `../../utils` too

Reply:
In Javascript, if you can import from direct files it actually helps a lot with circular dependency issues which cause modules to become undefined. It's not a hard rule I have, and I'm fine with developers doing either but I'll generally try to import from the file it's from.